README.md

```diff
@@ -26,39 +40,31 @@ pip install -r requirements.txt
 
-1. Import all [TC][2] and [incubator][3] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
-1. Run trace-server or project [1] below as Eclipse Application configured this way (or similar):
-   * `-Xms512m -Xmx1g`.
-   * Launch with `all workspace and enabled target plug-ins`.
-1. Browse [to here][4] ([swagger][5]) or so to generate server's TSP.
-1. Bring the resulting file over; e.g.: `mv ~/Downloads/openapi.yaml .`
+1. Import all [TC][tracecompass] and [incubator][incubator] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
+1. Open `traceserver.product` file in plug-in `org.eclipse.tracecompass.incubator.trace.server.product`.
+1. Click on the `Run` button on the top right corner of the opened `traceserver.product`.
+1. Browse [to here][apiyaml] ([swagger][swagger]) or so to generate server's TSP.
+1. The resulting file is stored in the user's Downloads directory; e.g.: `~/Downloads/openapi.yaml`
+1. Copy `~/Downloads/openapi.yaml` to this directory.
 1. Update the latter with its license information and remove extra information: `./openapi.py`
-1. The resulting git diff may then be pushed for review, at will.
-
-[1] org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core
-
-[2]: https://projects.eclipse.org/projects/tools.tracecompass/developer
-[3]: https://projects.eclipse.org/projects/tools.tracecompass.incubator/developer
-[4]: http://localhost:8080/tsp/api/openapi.yaml
-[5]: https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-configuration#openapiresource
-[6]: https://theia-ide.github.io/trace-server-protocol/
-[7]: https://theia-ide.github.io/trace-server-protocol/swagger/
-[8]: https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi
+1. The resulting diff between `API.yaml` and `openapi.yaml` can then be pushed for review. Note, that the order of fields, components etc. might be different everytime the API is generated using swagger-core. This is due to how swagger-core is implemented.
+1. Make sure to transfer the diffs to `API-proposed.yaml` as well.
+1. `openapi.yaml` should not be merged to the repository and can be deleted when not needed anymore.
+
+[tracecompass]: https://projects.eclipse.org/projects/tools.tracecompass/developer
+[incubator]: https://projects.eclipse.org/projects/tools.tracecompass.incubator/developer
+[apiyaml]: http://localhost:8080/tsp/api/openapi.yaml
+[swagger]: https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-configuration#openapiresource
+[tspGhPages]: https://theia-ide.github.io/trace-server-protocol/
+[vscodeOpenapi]: https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi
+[tspClient]: https://github.com/theia-ide/tsp-typescript-client
+[tcServer]: https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/
+[apiProposed]: https://theia-ide.github.io/trace-server-protocol/proposed/
```

Comment:
One last top commit could be added to this PR, to sort all these references alphabetically based on their tag. This would ease manual or visual look-ups, as well as consistent maintenance of their tags.

Reply:
will do